app/assets/javascripts/client/inventories/ManageInventoryAccessRequestsCtrl.js

```diff
@@ -0,0 +1,40 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient').controller('ManageInventoryAccessRequestsCtrl', ManageInventoryAccessRequestsCtrl); 
+
+  ManageInventoryAccessRequestsCtrl.$inject = ['$scope', 'InventoryAccessRequest'];
+
+  function ManageInventoryAccessRequestsCtrl($scope, InventoryAccessRequest) {
+    var vm = this;
+    vm.loadList = function() {
+      vm.list = InventoryAccessRequest.list({inventory_id: $scope.inventoryId});
+    };
+    vm.loadList();
+
+    vm.performAccessRequestAction = function(id, status) {
+      var params = { inventory_id: $scope.inventoryId, id: id };
+
+      InventoryAccessRequest.update(params, { status: status })
+      .$promise
+      .then(function() {
+        vm.loadList();
+      });
+    }
+
+    vm.denyRequest = function(id) {
+      if (confirm("Are you sure you want to deny this access request?")){
```

Comment:
To make this more testable, inject `$window` and then perform `$window.confirm`.